.flutter-plugins-dependencies

```diff
@@ -1 +1 @@
-{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"e2e","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/e2e-0.2.4+4/","dependencies":[]},{"name":"flutter_blue","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_blue-0.7.2/","dependencies":[]}],"android":[{"name":"e2e","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/e2e-0.2.4+4/","dependencies":[]},{"name":"flutter_blue","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_blue-0.7.2/","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"e2e","dependencies":[]},{"name":"flutter_blue","dependencies":[]}],"date_created":"2020-09-25 11:28:26.259992","version":"1.20.2"}
\ No newline at end of file
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"e2e","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/e2e-0.2.4+4/","dependencies":[]},{"name":"flutter_blue","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_blue-0.7.2/","dependencies":[]},{"name":"path_provider","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-0.4.1/","dependencies":[]},{"name":"permission_handler","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/permission_handler-5.0.1+1/","dependencies":[]}],"android":[{"name":"e2e","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/e2e-0.2.4+4/","dependencies":[]},{"name":"flutter_blue","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_blue-0.7.2/","dependencies":[]},{"name":"path_provider","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-0.4.1/","dependencies":[]},{"name":"permission_handler","path":"/home/abdullah/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/permission_handler-5.0.1+1/","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"e2e","dependencies":[]},{"name":"flutter_blue","dependencies":[]},{"name":"path_provider","dependencies":[]},{"name":"permission_handler","dependencies":[]}],"date_created":"2020-10-30 17:02:13.051116","version":"1.20.2"}
```

Comment:
Another auto generated file that shouldn't be in source control

Reply:
apparently it was being tracked since before